docs/pages/release_notes.md

```diff
@@ -1,6 +1,113 @@
 Release Notes {#releasenotes}
 ==============
 
+v3.5.0
+==============
+
+Major Updates
+-------------
+
+* ArrayFire now supports threaded applications
+  <sup>[1](https://github.com/arrayfire/arrayfire/pull/1706)</sup>
+* Added Canny edge detector
+  <sup>[1](https://github.com/arrayfire/arrayfire/pull/1743)</sup>
+* Added Sparse-Dense arithmetic operations
+  <sup>[1](https://github.com/arrayfire/arrayfire/pull/1696)</sup>
```

Comment:
@9prady9 the superscript is incremented when there are multiple links per item/bullet point.

Take a look at https://github.com/arrayfire/arrayfire/releases.